### PR TITLE
Document LRU memory usage

### PR DESCRIPTION
**Pull request by timbussmann**

### CODE REVIEW

File: nservicebus/recoverability/lru-memory-consumption.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ tags:
 reviewed: 2019-08-16
 ---
 
-MSMQ and SQL Server transport need to cache exceptions in memory for immediate retries. Therefore, exceptions with a large memory footprint can cause high memory usage of the NServiceBus process. The recoverability cache might require more resources when scaling out endpoints. This page describes how to solve issues caused by excessive memory consumption due to recoverability's caching mechanism.
+MSMQ and SQL Server transport needs to cache exceptions in memory to allow transactions to be cleared before executing recoverability policies. Therefore, exceptions with a large memory footprint can cause high memory usage of the NServiceBus process. Further, the recoverability cache might hold on to items longer when endpoints are scaled out. This page describes how to solve issues caused by excessive memory consumption due to this caching.
```
Comment (timbussmann) [inline nservicebus/recoverability/lru-memory-consumption.md]: I think `need` would be correct as we're mentioning two transports here (e.g. "they need").

Comment (timbussmann) [reply nservicebus/recoverability/lru-memory-consumption.md]: I think I'd use `Furthermore` instead of `Further` but that seems to be a detail.

Comment (andreasohlund) [reply nservicebus/recoverability/lru-memory-consumption.md]: Agree, updated (even though Further and Furthermore seems to be interchangeable https://fandom-grammar.livejournal.com/87140.html)

**Comment by andreasohlund** [comment]
Can you copy the snippets around?

>System.AggregateException : One or more errors occurred.
  ----> System.Exception : Could not render page: C:\BuildAgent\work\a48e3341ae42e493\nservicebus\recoverability\lru-memory-consumption.md
  ----> System.Exception : Failed to render article. PackageId:NServiceBus. Version:[6.0.0, 7.0.0).
  ----> System.Exception : MissingSnippet.
  Line: 10
  Key: dispose-large-exceptions

**Comment by timbussmann** [comment]
added the v6 snippet

File: nservicebus/recoverability/index.md [hunk 2]
```diff
@@ -100,3 +100,9 @@ The output in the log will be:
 snippet: RetryLogging
 
 partial: exceptionincluded
+
+
+## Recoverability memory consumption
+
+MSMQ and SQL Server transport need to cache exceptions in memory for retries. Therefore, exceptions with a large memory footprint can cause high memory usage of the NServiceBus process. NServiceBus can cache up to 1000 exceptions, capping the potential memory consumption to 1000 x `<exception size>`. Please refer to [this guide](/nservicebus/recoverability/lru-memory-consumption.md) to resolve problems due to execessive memory consumption.
```
Comment (andreasohlund) [inline nservicebus/recoverability/index.md]: @bording @SeanFeldman looking at the code we couldn't see that ASB is using a cache for the exceptions. Can you confirm that we haven't missed anything? (given that it does have support for atomic sends with receive)

Comment (SeanFeldman) [reply nservicebus/recoverability/index.md]: That's correct. ASB doesn't.

**Pull request merged**